### DISCUSSION TITLE
Using spaCy with AllenNLP Interpret or Textattack

### DISCUSSION THREAD

**Discussion by katarkor** [explosion/spaCy#5451] [New Features & Project Ideas]
Hi, I'd like to "attack" my spaCy models with either of the frameworks to better understand the results, but wasn't successful so far. Any tips? Are they at all compatible with spaCy (they're supposed to be model-agnostic)? If not: is there a spaCy way of doing something similar?

**Comment by svlandeg** [comment]
Hi @katarkor, I personally don't have any experience with either of these two frameworks, though I would be interested to see if/how that could work!

But I feel like this question is probably more suited for StackOverflow. You would reach a larger community (including people who know/contribute to the code by AllenNLP) and it helps us keeping this issue tracker focused on bugs in spaCy. I would advise to provide a working code snippet for your SO post, that shows what you've tried, allows people to replicate your results, and shows what exactly is going wrong. Hopefully someone will be able to help you further there.

**Comment by svlandeg** [comment]
Linking to https://stackoverflow.com/questions/61889477/using-spacy-models-with-allennlp-interpret-or-textattack for completeness :-)
